packages/cardhost/app/components/code-editor.js

```diff
@@ -27,8 +28,8 @@ export default class CodeEditor extends Component {
 
   constructor(...args) {
     super(...args);
-
-    this.resizable = (this.args.resizable === true) ? true : false;
+    let notTesting = ENV.environment !== 'test'; // disable resize loops in testing
+    this.resizable = (this.args.resizable === true && notTesting) ? true : false;
```

Comment:
I'm generally not a fan of putting test code into implementation code, but in this case I'm not sure how to disable the timer when in test. This is for an acceptance test right?

Reply:
Surprisingly, this works:

```
  hooks.beforeEach(function () {
    this.owner.lookup('service:data')._clearCache();
    // any time you visit the editor page, you need to set resizable to
    // false, or tests will time out.
    this.owner.lookup('controller:cards.view').resizable = false;
  });
```